src/components/Pricing/PlanComparison/index.tsx

```diff
@@ -235,7 +226,7 @@ const sections = [
             },
             {
                 name: 'Zapier',
-                tiers: { 'PostHog Cloud': true, 'Open source': true, Scale: true, Enterprise: true },
+                tiers: { 'PostHog Cloud': false, 'Open source': true, Scale: true, Enterprise: true },
```

Comment:
We no longer support Zapier on Cloud?

Reply:
Oops, I mixed something up when rebasing